src/NUnitFramework/framework/Constraints/ThrowsNothingConstraint.cs

```diff
@@ -50,23 +50,9 @@ public override string Description
         /// <returns>True if no exception is thrown, otherwise false</returns>
         public override ConstraintResult ApplyTo<TActual>(TActual actual)
         {
-            //TestDelegate code = actual as TestDelegate;
-            //if (code == null)
-            //    throw new ArgumentException("The actual value must be a TestDelegate", "actual");
-
-            //caughtException = null;
-
-            //try
-            //{
-            //    code();
-            //}
-            //catch (Exception ex)
-            //{
-            //    caughtException = ex;
-            //}
-
```

Comment:
I took the liberty of deleting old commented code, let me know if I should put it back